docs/user/installation_and_configuration.rst

```diff
@@ -44,6 +44,17 @@ Versions and Downloads
 
 * Source: https://github.com/locationtech/geomesa/archive/master.tar.gz
 
+**Fully Eclipse IP reviewed release**: |eclipse_release|
```

Comment:
[picky] The wording here seems a bit funky, even though I know what you're going for.  Perhaps something like, "Most recent release for which Eclipse has fully vetted IP"?  Or, "Most recent release whose IP has been fully reviewed by Eclipse"?  I dunno'.